AK/Variant.h

```diff
@@ -499,9 +499,24 @@ struct Variant
 template<typename... Ts>
```

Comment:
Commit message: please use punctuation and "IDL" instead of "idl". You can let the commit message auto-wrap at 72 characters, you don't need to do that manually.

Reply:
I changed the commit message, but since I am not using CLion but VSCode, (which has no auto wrap feature, at least that I know of) I did the wrap manually anyway, but closer to the 72 limit then before.